tools/clang/unittests/HLSLExec/HlslExecTestUtils.cpp

```diff
@@ -212,6 +213,53 @@ static bool createDevice(
   return true;
 }
 
+// Read a resource embedded into a dll via an .rc file and wrap it in a DXC
+// read-only stream
+void readEmbeddedHlslDataIntoNewStream(
+    LPCWSTR ResourceName, // Resource name in rc file. e.g. L"LongVectorOp"
+    IStream **TestXML, dxc::SpecificDllLoader &Support) {
+
+  DllWrapper Dll;
+  Dll.Module = LoadLibraryEx(TEXT("ExecHLSLTests.dll"), nullptr,
```

Comment:
@damyanp You mentioned the hard coded module name would be a problem here. I think we can change to using the name from GetModuleFileName to handle the tests have a different dll name?